cmd/component.go

```diff
@@ -3,19 +3,26 @@ package cmd
 import (
 	"fmt"
 
+	"os"
+
 	"github.com/redhat-developer/odo/pkg/application"
 	"github.com/redhat-developer/odo/pkg/component"
 	"github.com/redhat-developer/odo/pkg/project"
 	log "github.com/sirupsen/logrus"
 	"github.com/spf13/cobra"
-	"os"
 )
 
 // componentCmd represents the component command
 var componentCmd = &cobra.Command{
 	Use:   "component",
 	Short: "Components of application.",
+	Example: `  # Get current component,
+  odo component 
+  # Set nodejs to current component,
```

Comment:
can you please add an empty line between each example?
It would be nice to keep it consistent with other places like `odo create -h`